vendor/bat-native-ads/src/bat/ads/internal/reports.cc

```diff
@@ -31,7 +32,8 @@ std::string Reports::GenerateAdNotificationEventReport(
   rapidjson::StringBuffer buffer;
   rapidjson::Writer<rapidjson::StringBuffer> writer(buffer);
 
-  const std::string timestamp = Time::Timestamp();
+  const base::Time time = base::Time::Now();
+  const std::string timestamp = FriendlyDateAndTime(time, false);

```

Comment:
I suppose this isn't really a "timestamp" anymore, but rather a formatted date string, but it looks like renaming the variable/JSON field would be painful.

Reply:
Logs are being refactored as part of https://github.com/brave/brave-browser/issues/6845 and https://github.com/brave/brave-browser/issues/9533 so reports will be deprecated